src/documentation/pages/components/drawer/_properties.vue

```diff
@@ -0,0 +1,382 @@
+                  | Lets the developer refer the target programmatically instead of by data-target attribute.
+                  | It accepts Element object and xpath selector string.
+            tr
+              td
+                div
+                  code= 'preventAutoHide'
+              td
+                div
+                  code= 'false'
+              td
+                div
+                  | Set <code>preventAutoHide</code> to <code>true</code> to prevent hidding the drawer when clicking
+                  | outside the drawer.
+
+    h3.mt--2 Methods
+    section.chi-table.chi-table__methods.-bordered.-my--3
+      div
+        table.-text(cellpadding='0' cellspacing='0')
+          thead
+            tr
+              th
+                div Method
+              th
+                div Description
+              th
+                div Returns
+          tbody
+            tr
+              td
+                div
+                  code= 'hide() => void'
+              td
+                div
+                  | Hides the drawer
+              td
+                div
+                  | Type: void
+            tr
+              td
+                div
+                  code= 'show() => void'
+              td
+                div
+                  | Shows the drawer
+              td
+                div
+                  | Type: void
+            tr
+              td
+                div
+                  code= 'toggle() => void'
+              td
+                div
+                  | Toggles active state (show/hide)
+              td
+                div
+                  | Type: void
+
+    h3 Events
+    section.chi-table.-bordered.-my--3
+      table(cellpadding='0' cellspacing='0')
+        thead
+          tr
+            th
+              div Event
+            th
+              div Description
+        tbody
+          tr
+            td
+              div
+                code chiDrawerShow
+            td
+              div Drawer show has been triggered, but the showing animation has not started yet.
+          tr
+            td
+              div
+                code chiDrawerHide
+            td
+              div Drawer hide has been triggered, but the closing animation has not started yet.
+          tr
+            td
+              div
+                code chiDrawerShown
+            td
+              | Drawer has been shown to the user and is fully visible. The animation has completed.
+          tr
+            td
+              div
+                code chiDrawerHidden
+            td
+              div
+                | Drawer has been hidden to the user. The animation has completed.
+
+    h3 Preventing memory leaks
+    p.-text
+      | Drawer components have a dispose function to free all resources attached to the element, such as event listeners
+      | and object data. You should call this method when you want to remove the component.
+      pre.language-javascript
```

Comment:
Some padding can be added between the description text and code snippet.
<img width="563" alt="padding" src="https://user-images.githubusercontent.com/92325462/160314292-c4171f3c-0f0e-429b-9486-58c8f21b5a63.png">

Reply:
The error here was that the `pre` element was inside the `p`.

Now it's fixed, nice catch!